components/library/poppler/Makefile

```diff
@@ -54,17 +57,14 @@ CONFIGURE_OPTIONS += CPPFLAGS="$(CPP_XPG6MODE) $(CPP_LARGEFILES)"
 CONFIGURE_OPTIONS += CXX="$(CXX) $($(COMPILER)_NORUNPATH)"
 CONFIGURE_OPTIONS += --enable-poppler-glib
 CONFIGURE_OPTIONS += --disable-poppler-qt4
-CONFIGURE_OPTIONS += --disable-poppler-qt5
+CONFIGURE_OPTIONS += --enable-poppler-qt5
 CONFIGURE_OPTIONS += --enable-zlib
 CONFIGURE_OPTIONS += --enable-xpdf-headers
 
 COMPONENT_BUILD_ENV += CC="$(CC)"
 COMPONENT_BUILD_ENV += CXX="$(CXX)"
 COMPONENT_BUILD_ENV += CFLAGS="$(CFLAGS)"
 
-build:		$(BUILD_32_and_64)
-
-install:	$(INSTALL_32_and_64)
 
 test:		$(NO_TESTS)
 
```

Comment:
Remove the test: $(NO_TESTS) here and replace it with
TEST_TARGET= $(NO_TESTS)
(and preferably move it up before the line that includes common.mk).